MAUI/Cartesian-Charts/Customize-axis-label-style-using-labelCreated-event.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: Customize axis label style using labelCreated event | Syncfusion
+description: Learn here all about to customize axis label style using labelCreated event in chart axis in Syncfusion .NET MAUI Chart (SfCartesianChart) control.
+platform: maui
+control: SfCartesianChart
+documentation: ug
+---
+
+# Customize axis label style using labelCreated event
+
+The [`LabelCreated`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.ChartAxis.html#Syncfusion_Maui_Charts_ChartAxis_LabelCreated) event is triggered upon label creation in a chart axis, providing the option to customize axis labels style.
+The [LabelStyle](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.ChartAxisLabelEventArgs.html#Syncfusion_Maui_Charts_ChartAxisLabelEventArgs_LabelStyle) property is available to customize the appearance of axis label based on condition.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+    <chart:SfCartesianChart>
+        . . .
+        <chart:SfCartesianChart.XAxes>
+            <chart:NumericalAxis LabelCreated="XAxes_LabelCreated"/>
+        </chart:SfCartesianChart.XAxes>
+        . . .
+    </chart:SfCartesianChart>
+
+{% endhighlight %}
+
+{% tabs %}
+
+{% highlight c# %}
+
+        //Customize the style of chart axis labels and edit the axis labels using the labelCreated event.
+        private void XAxes_LabelCreated(object sender, ChartAxisLabelEventArgs e)
```

Comment:
This can be achieved through, chart axis label style, why need to come label created event? Not you understand the usage of this feature.